bindings/pyroot_experimental/cppyy/CPyCppyy/.gitignore

```diff
@@ -0,0 +1,14 @@
+# python compiled files
```

Comment:
If those end up here we have a (build system) problem. Can we get rid of these gitignores, or are they coming from upstream and Wim supports in-source builds?

Reply:
They are coming from upstream, and we want to take the code as is to avoid having to keep track of our own changes. This gitignore should be ok, no build files will end up in these directories.